plugins/UsersManager/Validators/Login.php

```diff
@@ -18,8 +18,8 @@
 
 class Login extends BaseValidator
 {
-    const loginMinimumLength = 2;
-    const loginMaximumLength = 100;
+    const LOGIN_MIN_LENGTH = 2;
```

Comment:
BC, someone might be using this 🤷

Reply:
So as long as we mention it in the changelog for Matomo 5, we're good.

Reply:
That validator has been introduced "silently" last year and is not marked as public `@api`, so we could make such changes even without mentioning it in the changelog.

Reply:
Again, devil's advocate here — should we then mark it as @internal (or change the visibility as `private` if possible?)